src/compile/scale/range.ts

```diff
@@ -214,6 +214,10 @@ export function defaultRange(
 
 function sizeRangeMin(mark: Mark, zero: boolean, config: Config) {
   if (zero) {
+    if (util.contains(['point', 'square', 'circle'], mark)) {
+      // the smallest size for points should not be zero as the marks become invisible
```

Comment:
But this also make scale incorrect. 

I'd say, if you have "zero" value, better not use size.

Reply:
Maybe 0.01 or 0.1 is palatable

Reply:
Try it out, you still have white inside the circle but it looks worse.

Reply:
Already tried and it's terrible.  In general, using size to encode field with zero values is a bad idea anyway, so I don't think we should make size less accurate for this bad use case.

Reply:
The change in accuracy is small but https://github.com/vega/vega-lite/issues/3488 seems problematic. If you disagree, we should close the issue and this PR.

Reply:
Let's try production rule? https://github.com/vega/vega-lite/issues/3488#issuecomment-383388466